### PR TITLE
Tickets/dm 38516

### PR DESCRIPTION
**Pull request by teweitsai**
* Support the mypy.

### CODE REVIEW

File: python/lsst/ts/m2gui/__init__.py [hunk 1]
```diff
@@ -29,49 +29,17 @@
...
-from .fault_manager import FaultManager
-from .actuator_force import ActuatorForce
-from .force_error_tangent import ForceErrorTangent
-from .utility_monitor import UtilityMonitor
-from .config import Config
-from .model import Model
-from .control_tabs import ControlTabs
-from .main_window import MainWindow
-from .application import run_application
+from .enums import *
```
Comment (tribeiro) [inline python/lsst/ts/m2gui/__init__.py:32]: you didn't mentioned this change in the version history.

File: python/lsst/ts/m2gui/control_tabs.py [hunk 2]
```diff
@@ -53,9 +57,9 @@ class ControlTabs(object):
         Layout.
     """
 
-    def __init__(self, model):
+    def __init__(self, model: Model) -> None:
         # List of the control tables
-        self._tabs = list()
+        self._tabs: typing.List[TabDefault] = list()
```
Comment (tribeiro) [inline python/lsst/ts/m2gui/control_tabs.py:62]: I think you could use `list[TabDefault]`. My impression is that `typing.List` is supposed to be deprecated in favor of `list`.

File: python/lsst/ts/m2gui/control_tabs.py [hunk 3]
```diff
@@ -121,9 +125,13 @@ async def _callback_show(self, item):
...
         control_tab = self._get_control_tab(item.text())
-        control_tab.show()
 
-    def get_tab(self, name):
+        if control_tab is not None:
+            control_tab.show()
+
+    def get_tab(
+        self, name: str
+    ) -> typing.Tuple[QListWidgetItem | None, TabDefault | None]:
```
Comment (tribeiro) [inline python/lsst/ts/m2gui/control_tabs.py:134]: I think you can use `tuple` instead of `typing.Tuple`, same argument as the `list`...

**Review by tribeiro — APPROVED** [review]
Looks good.. A couple things to note though, 

I think you can use `list` and `tuple` instead of `typing.List` and `typing.Tuple`. Also, I noticed you changed the import statements in some of the `__init__` files, but these changes are not mentioned in the version history. Maybe it is fine to leave them out, up to you.

File: python/lsst/ts/m2gui/control_tabs.py (hunk 2 above)
Comment (teweitsai) [reply python/lsst/ts/m2gui/control_tabs.py]: Fixed in the commit: [b8ec396](https://github.com/lsst-ts/ts_m2gui/pull/35/commits/b8ec3962d7117af883c72fd25835442da89caf3d). Thanks!

File: python/lsst/ts/m2gui/control_tabs.py (hunk 3 above)
Comment (teweitsai) [reply python/lsst/ts/m2gui/control_tabs.py]: Fixed in the commit: [b8ec396](https://github.com/lsst-ts/ts_m2gui/pull/35/commits/b8ec3962d7117af883c72fd25835442da89caf3d). Thanks!

**Pull request merged**